src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -230,7 +317,7 @@ private boolean isApiFoundIntheList(String url) {
 		boolean isMatch=false;
 		Set<String> apiListRegex = apiListRegexWithDebugMsg.keySet();
 		for (String api : apiListRegex) {
-			if(url.matches(props.getString(api))) {
+			if(url.matches(appExtPropertiesFile.getProperty(api))) {
```

Comment:
Get string is used in the properties bundle object. Get property is the method used in the property object.